odoo/tools/translate.py

```diff
@@ -1105,6 +1137,9 @@ def _export_translatable_resources(self):
                 for fname in fnmatch.filter(files, '*.py'):
                     self._babel_extract_terms(fname, path, root,
                                               extract_keywords={'_': None, '_lt': None})
+                if fnmatch.fnmatch(root, '*/data/*'):
+                    for fname in fnmatch.filter(files, "*_dashboard.json"):
```

Comment:
To reviewers: we should probably be more selective concerning what json files are considered.
Maybe rename all spreadsheet data files with `*.spreadsheet.json` ? in `master` ?